tests/plugins/keyword_test.py

```diff
@@ -10,6 +13,8 @@
 LETTER_SECRET = 'A,.:-¨@*¿?!'
 SYMBOL_SECRET = ',.:-¨@*¿?!'
 
+LONG_LINE = '<img src="data:image/png;base64,{}">'.format(base64.b64encode((str(randint(0, 9)) * 30500).encode()))  # noqa: E501
```

Comment:
```suggestion
LONG_LINE = '<img src="data:image/png;base64,{}\n"\n>'.format(base64.b64encode((str(randint(0, 9)) * 30500).encode()))  # noqa: E501
```

The new lines *shouldn't* do anything, but it will screw up regexes that assume properly terminated quotes and HTML tags.

Reply:
I'm sorry. I didn't see your comment the last day. Done!